index.html

```diff
@@ -132,5 +123,251 @@
     
         </section>
     </header>
+
+    <section class="works">
```

Comment:
- [ ] As requested by the previous reviewer, kindly change the background color of the work section to match the given template. You are expected to stick to design as much as possible. The background color on the figma template is `#e5e5e5` and your background is white.
### Figma
![image](https://user-images.githubusercontent.com/94551484/221348438-ef8bcbd8-d686-4efb-8658-93fa7b5d4d09.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/221348530-6c3d1d63-ddf3-47d5-bc64-01a1ec45d7b1.png)